corporate/urls.py

```diff
@@ -170,9 +173,27 @@
     path("realm/<realm_uuid>/billing", remote_billing_page_realm, name="remote_billing_page_realm"),
     path("server/<server_uuid>/", remote_billing_page_server, name="remote_billing_page_server"),
     path("realm/<realm_uuid>/upgrade", remote_realm_upgrade_page, name="remote_realm_upgrade_page"),
+    path(
+        "realm/<realm_uuid>/sponsorship",
+        remote_realm_sponsorship_page,
+        name="remote_realm_sponsorship_page",
+    ),
+    path(
+        "server/<server_uuid>/sponsorship",
+        remote_server_sponsorship_page,
+        name="remote_server_sponsorship_page",
+    ),
     path(
         "serverlogin/",
         remote_billing_legacy_server_login,
         name="remote_billing_legacy_server_login",
     ),
+    # Remote variants of above API endpoints.
+    path("json/realm/<realm_uuid>/sponsorship", remote_realm_sponsorship),
+    path("json/server/<server_uuid>/sponsorship", remote_server_sponsorship),
```

Comment:
Is this comment correct? It's not clear to me which endpoints `of above API endpoints.` refers to specifically